### ISSUE TITLE
Translation: automatically update crowdin project

### ISSUE THREAD

**Issue by getdatakick** [thirtybees/ThirtyBees#1357]
Implement task to automatically extract translations from *core* and all *native modules*, and upload changes to crowdin

**Comment by comradekingu** [comment]
Why go down that road instead of just using Weblate right away?

**Comment by getdatakick** [comment]
Good idea @comradekingu, thanks for the suggestion. I will investigate how hard it would be to implement this

**Comment by cienislaw** [comment]
I'm confused - its about translations or English source texts? and in what way Weblate would be better than Crowdin?

**Comment by comradekingu** [comment]
@cienislaw Both are translation platforms.
Crowdin is spy on the user, non-free, as-a-service only product, and requires libre projects to be non-commercial for free of charge hosting. There is no community.
Weblate is copylefted libre software, has has good terms for the "Hosted Weblate serveice", it has free of charge hosting for libre projects. It has a great community.

In Weblate you can see multiple languages at once, easily find failing checks, and skip between many strings at once, while seeing both the translation and source strings. The list goes on forever.
I like the Crowdin dictionary a bit more.

**Comment by cienislaw** [comment]
@comradekingu thank you for explanation.

@getdatakick for me having current English source and latest translation distribution is more important than where it has to be translated.
